README.md

```diff
@@ -65,10 +65,10 @@ ESP (Enterprise Services Platform) provides an easy to use interface to access E
 
 The following links describe the structure of the system and detail some of the key components:
 
-* [An overview of workunits and the different stages in executing a query](https://github.com/hpcc-systems/HPCC-Platform/blob/master/devdoc/WorkUnits.rst)
-* [An introduction to the code generator - eclcc](https://github.com/hpcc-systems/HPCC-Platform/blob/master/devdoc/CodeGenerator.rst)
-* [The memory manager used by roxie and thor](https://github.com/hpcc-systems/HPCC-Platform/blob/master/devdoc/MemoryManager.rst)
-* [The structure of the initialization scripts](https://github.com/hpcc-systems/HPCC-Platform/blob/master/initfiles/DOCUMENTATION.rst)
-* [Outline of ecl-bundle](https://github.com/hpcc-systems/HPCC-Platform/blob/master/ecl/ecl-bundle/DOCUMENTATION.rst)
-* [The structure and some details of the cmake files](https://github.com/hpcc-systems/HPCC-Platform/blob/master/cmake_modules/DOCUMENTATION.rst)
-* [Building the documentation](https://github.com/hpcc-systems/HPCC-Platform/blob/master/docs/DOCUMENTATION.rst)
+* [An overview of workunits and the different stages in executing a query](https://hpcc-systems.github.io/HPCC-Platform/devdoc/Workunits.html)
+* [An introduction to the code generator - eclcc](https://hpcc-systems.github.io/HPCC-Platform/devdoc/CodeGenerator.html)
```

Comment:
html -> md for all the examples in this file?  Or have html files been generated from the md?

Reply:
This README.md is the one that is displayed by default in GitHub, so the links need to be full to get "into" the rendered docs